models/track_and_reid_model.py

```diff
@@ -1,28 +1,15 @@
-import csv
-import os
-import pickle
-import tempfile
 import time
 from argparse import ArgumentParser, REMAINDER
 import sys
-from collections import defaultdict, Counter
-from torchvision.transforms import transforms
+from collections import defaultdict
+import matplotlib.pyplot as plt
 from PIL import Image
-import cv2
 import pandas as pd
-from matplotlib import pyplot as plt
-import tqdm
-import torch
-import numpy as np
 import mmcv
 import torch.nn.functional as F
 from DataProcessing.DB.dal import *
 from DataProcessing.dataProcessingConstants import ID_TO_NAME, NAME_TO_ID
-from FaceDetection.arcface import ArcFace, GALLERY_NO_UNKNOWNS, GPIDS_NO_UNKNOWNS, is_img, GALLERY_PKL_PATH, \
-    GPIDS_PKL_PATH
-from FaceDetection.augmentions import normalize_image
```

Comment:
Sure about all these import removals? there are a lot of them here

Reply:
Yep, FaceDetection class is dead

Reply:
<img width="866" alt="image" src="https://user-images.githubusercontent.com/61732335/193446635-baa04eb1-207b-40b5-a8bf-ee784f94b0d7.png">
I mean all the other imports like torch, tqdm, numpy etc.

Reply:
You are correct. Things work now because we import them through other models (which should not be the case), I'll correct this.